### PR TITLE
Package fadbadml.0.1

### PR DESCRIPTION
**Pull request by ismailbennani**
### `fadbadml.0.1`
FADBAD++ for OCaml

---
* Homepage: https://github.com/ismailbennani/FADBADml
* Source repo: git+https://github.com/ismailbennani/FADBADml
* Bug tracker: https://github.com/ismailbennani/FADBADml/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/fadbadml/fadbadml.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+homepage: "https://github.com/ismailbennani/FADBADml"
+bug-reports: "https://github.com/ismailbennani/FADBADml/issues"
+depends: [ "ocaml" { >= "4.08" } "ocamlfind" ]
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make "byte"] { !ocaml-native }
+  [make "opt"] { ocaml-native }
+]
+install: [make "install"]
+dev-repo: "git+https://github.com/ismailbennani/FADBADml"
```
Comment (kit-ty-kate) [inline packages/fadbadml/fadbadml.0.1/opam]: This repository does not exist

File: packages/fadbadml/fadbadml.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "ismailbennani <ismail.lahkim.bennani@ens.fr>"
+]
+authors: [
+  "francois-bidet <fbidet@lix.polytechnique.fr>"
+  "ismailbennani <ismail.lahkim.bennani@ens.fr>"
+]
+license: "CeCILL-C"
+homepage: "https://github.com/ismailbennani/FADBADml"
+bug-reports: "https://github.com/ismailbennani/FADBADml/issues"
+depends: [ "ocaml" { >= "4.08" } "ocamlfind" ]
```
Comment (kit-ty-kate) [inline packages/fadbadml/fadbadml.0.1/opam]:
```suggestion
depends: [
  "ocaml" {>= "4.08"}
  "ocamlfind" {build}
]
```
ocamlfind should be tagged as a build dependency

File: packages/fadbadml/fadbadml.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make "byte"] { !ocaml-native }
+  [make "opt"] { ocaml-native }
```
Comment (kit-ty-kate) [inline packages/fadbadml/fadbadml.0.1/opam]: The `ocaml-native` variable does not exist. I think what you meant is `ocaml:native`. However according the the `install` rule in your Makefile, these won't have any effect anyway as it relies on the `lib` rule. Those two lines should be replaced by a simple `[make "lib"]`.

File: packages/fadbadml/fadbadml.0.1/opam (hunk 1 above)
Comment (ismailbennani) [reply packages/fadbadml/fadbadml.0.1/opam]: Fixed, thanks

File: packages/fadbadml/fadbadml.0.1/opam (hunk 3 above)
Comment (ismailbennani) [reply packages/fadbadml/fadbadml.0.1/opam]: Done, thanks

File: packages/fadbadml/fadbadml.0.1/opam (hunk 2 above)
Comment (ismailbennani) [reply packages/fadbadml/fadbadml.0.1/opam]: thanks !

**Comment by mseri** [comment]
It looks like the source tarball has changed in the meanwhile:

```
[ERROR] The sources of the following couldn't be obtained, aborting:
4284          - fadbadml.0.1:
4285            https://github.com/fadbadml-dev/FADBADml/archive/0.1.tar.gz (Bad
4286        checksum,
4287            expected md5=1499620e66d402f24d3c0413f1f0eef7)
```

**Comment by ismailbennani** [comment]
I ran opam-publish again, the checksums should be ok now !

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**